src/main.rs

```diff
@@ -11,121 +10,126 @@ use hickory_client::{
     rr::{rdata::TXT, Name, RData, Record, RecordType},
 };
 
-use log::{debug, info, trace};
-extern crate pretty_env_logger;
-#[macro_use]
-extern crate log;
+use tracing::{debug, info, trace, warn, error};
+use tracing_subscriber::{EnvFilter, FmtSubscriber};
+
+use once_cell::sync::OnceCell;
 
 mod cli;
 mod config;
 mod happy;
```

Comment:
I wouldn't call it "happy" anymore tbqh, and I'd maybe even inline this.

Reply:
inlined ^^